submissions/madmaxWMFU/a-tiny-JS-world-OOP/index.js

```diff
@@ -0,0 +1,67 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+class Objects {
```

Comment:
We must define classes with singular, but `Object` name is reserved. Maybe use another common name? `Inhabitant` will be better.